src/data_structure/temporal_structure.jl

```diff
@@ -283,6 +283,7 @@ function generate_temporal_structure!(m::Model)
     _generate_current_window!(m::Model)
     _generate_time_slice!(m::Model)
     _generate_time_slice_relationships!(m::Model)
+    rep_time_slice_mapping(m::Model)
```

Comment:
I think, for consistency this should be called `_generate_representative_time_slice_mapping()`.

Also, I'm not sure I'm a fan of abbreviating `representative` to `rep` in general?